src/style.css

```diff
@@ -0,0 +1,39 @@
+.body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
```

Comment:
`flex-direction: row` - this is the default. no need to set it explicitly
`flex-wrap` is not needed here